File: cours-filters-java/src/test/java/fr/isima/controllers/GoodByeControllerTest.java
package fr.isima.controllers;

import org.junit.Before;
import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class GoodByeControllerTest {

    private GoodByeController goodByeController;

    @Before
    public void setUp() throws Exception {
        this.goodByeController = new GoodByeController();
    }

    @Test
    public void goodbye_returnGoodbyeString() {
        assertThat(this.goodByeController.goodBye()).isEqualTo("GoodBye !");
    }
}

File: cours-filters-kotlin/src/test/kotlin/controllers/GoodByeControllerTest.kt
package controllers

import fr.isima.controllers.GoodByeController
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class GoodByeControllerTest {

    private val goodByeController = GoodByeController()

    @Test
    fun goodbye_returnGoodbyeString() {
        assertThat(this.goodByeController.goodBye()).isEqualTo("GoodBye !")
    }
}